integration/flink/flink2/src/test/java/io/openlineage/flink/FlinkContainerUtils.java

```diff
@@ -0,0 +1,252 @@
+        .withEnv("SCHEMA_REGISTRY_HOST_NAME", "schema-registry")
+        .withEnv("SCHEMA_REGISTRY_LISTENERS", "http://schema-registry:8081,http://0.0.0.0:28081")
+        .withEnv("SCHEMA_REGISTRY_LOG4J_ROOT_LOGLEVEL", "WARN")
+        .dependsOn(startable);
+  }
+
+  static GenericContainer<?> makeKafkaContainer(Network network, Startable zookeeper) {
+    return genericContainer(network, KAFKA_IMAGE, "kafka-host")
+        .withExposedPorts(9092, 19092)
+        .withEnv(
+            "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP",
+            "LISTENER_DOCKER_INTERNAL:PLAINTEXT,LISTENER_DOCKER_EXTERNAL:PLAINTEXT")
+        .withEnv(
+            "KAFKA_LISTENERS",
+            "LISTENER_DOCKER_INTERNAL://kafka-host:9092,LISTENER_DOCKER_EXTERNAL://127.0.0.1:19092")
+        .withEnv(
+            "KAFKA_ADVERTISED_LISTENERS",
+            "LISTENER_DOCKER_INTERNAL://kafka-host:9092,LISTENER_DOCKER_EXTERNAL://127.0.0.1:19092")
+        .withEnv("KAFKA_INTER_BROKER_LISTENER_NAME", "LISTENER_DOCKER_INTERNAL")
+        .withEnv("KAFKA_ZOOKEEPER_CONNECT", "zookeeper:2181")
+        .withEnv("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "1")
+        .withEnv("KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS", "0")
+        .withEnv("KAFKA_TRANSACTION_STATE_LOG_MIN_ISR", "1")
+        .withEnv("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "1")
+        .withEnv("TOPIC_AUTO_CREATE", "true")
+        .withEnv("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "1")
+        .withEnv("KAFKA_BROKER_ID", "1")
+        .withEnv(
+            "KAFKA_LOG4J_LOGGERS",
+            "kafka.controller=WARN,kafka.producer.async.DefaultEventHandler=WARN,state.change.logger=INFO")
+        .withEnv("LOG4J_LOGGER_KAFKA", "WARN")
+        .dependsOn(zookeeper);
+  }
+
+  @SneakyThrows
+  static GenericContainer<?> makeGenerateEventsContainer(Network network, Startable initTopics) {
+    return genericContainer(network, SCHEMA_REGISTRY_IMAGE, "generate-events")
+        // TODO: to be reused later when adding additional tests
+        .withCopyFileToContainer(
+            MountableFile.forHostPath(Resources.getResource("InputEvent.avsc").getPath()),
+            "/tmp/InputEvent.avsc")
+        //        .withCopyFileToContainer(
+        //
+        // MountableFile.forHostPath(Resources.getResource("InputEvent.proto").getPath()),
+        //            "/tmp/InputEvent.proto")
+        //        .withCopyFileToContainer(
+        //            MountableFile.forHostPath(Resources.getResource("events.json").getPath()),
+        //            "/tmp/events.json")
+        //        .withCopyFileToContainer(
+        //
+        // MountableFile.forHostPath(Resources.getResource("events_proto.json").getPath()),
+        //            "/tmp/events_proto.json")
+        //        .withCommand(
+        //            "/bin/bash",
+        //            "-c",
+        //            Resources.toString(Resources.getResource("generate_events.sh"),
+        // StandardCharsets.UTF_8))
```

Comment:
Do we need those?

Reply:
we will need it in future, we can remove it for now.